packages/SymfonyEventDispatcher/tests/Adapter/Nette/DI/LazyEventSubscribersTest.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\SymfonyEventDispatcher\Tests\Adapter\Nette\DI;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symplify\SymfonyEventDispatcher\Tests\Adapter\Nette\ContainerFactory;
+use Throwable;
+
+final class LazyEventSubscribersTest extends TestCase
+{
+    public function test(): void
+    {
+        $container = (new ContainerFactory)->createWithConfig(__DIR__ . '/../config/lazySubscribers.neon');
+
+        /** @var EventDispatcherInterface $eventDispatcher */
+        $eventDispatcher = $container->getByType(EventDispatcherInterface::class);
+
+        // Make sure the subscriber is initialized lazily.
+        $this->assertFalse($container->isCreated('subscriber'));
```

Comment:
Could you move this to standalone method with descriptive name (instead of comment)?